README.md

```diff
@@ -4,8 +4,6 @@ Application demonstrating a [WebAuthn](https://en.wikipedia.org/wiki/WebAuthn) p
 
 If you want to see an implementation of WebAuthn as a second factor authenticator in a Rails application, you can check it in [webauthn-2fa-rails-demo](https://github.com/cedarcode/webauthn-2fa-rails-demo).
 
-[![Travis](https://img.shields.io/travis/cedarcode/webauthn-rails-demo-app/master.svg?style=flat-square)](https://travis-ci.org/cedarcode/webauthn-rails-demo-app)
```

Comment:
Should we use the one from Github Actions?

```
[![Build](https://github.com/cedarcode/webauthn-rails-demo-app/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/cedarcode/webauthn-rails-demo-app/actions/workflows/ci.yml)
```

Reply:
not sure, up to you

i'm not really sure how useful this actually is, because the CI should always be passing in order to merge to main?